### ISSUE TITLE
Route animations mess up on iOS Chrome

### ISSUE THREAD

**Issue by MagnusBrzenk** [tomastrajan/angular-ngrx-material-starter#451]
## Minimal reproduction of the bug with instructions:
Changing pages on iOS Chrome results in jerky animations.

## Expected behavior:
Smooth animations.

## I would be willing to submit a PR to fix this issue:

[ ] Yes (Assistance is provided if you need help submitting a pull request)  
[X] No

**Comment by tomastrajan** [comment]
@MagnusBrzenk thanks for reporting the issue. As a quick workaround , it is possible to disable animations in the settings, and potential in code if you're using it for your own app.

**Comment by MagnusBrzenk** [comment]
Thanks @tomastrajan. I ended up doing some research into layout and angular animations and ended up building [my own boilerplate](https://github.com/MagnusBrzenk/ng7-material-boilerplate) that borrows from this repo and [this one](https://github.com/CanKattwinkel/material-2-sticky-footer-mat-sidenav). So if you're having trouble with layout/animations, you might want to check out the simplified approach shown there. It was also important to my goals to have a sticky footer, which isn't a constraint here.

(@tomastrajan: I think the approach to layout/animations that I went for in this repo is too different to try and implement here via a pull request, so I figured the best I could do is link to what I've come up with, and maybe that can give you ideas.)
